lib/file.coffee

```diff
@@ -43,6 +40,12 @@ class File
   onDidStatusChange: (callback) ->
     @emitter.on('did-status-change', callback)
 
+  loadRealPath: ->
```

Comment:
You are correct, int this package, promises aren't used anywhere else and this PR introduces with requiring `Q`
personally, I don't like promises, but I wanted to be consistent with Atom's core usages of promises.

Since the package was doing most of the code synchronously, there wasn't a clear cut for me whether to use promises or callbacks.

I don't mind refactoring it back to callback if you think including q and using promises is confusing for this package